osquery/filesystem/filesystem.cpp

```diff
@@ -99,6 +101,15 @@ struct OpenReadableFile : private boost::noncopyable {
   bool blocking_io;
 };
 
+void initializeFilesystemAPILocale() {
+#if defined(WIN32)
+  setlocale(LC_ALL, ".UTF-8");
```

Comment:
Does this set the default, or does it have the effect of overriding whatever is already set?

Reply:
This will override the default value, so that the internal conversion from UTF-16 (inside boost::filesytem::path, std::filesystem::path) will correctly target UTF-8

Reply:
Okay, I don't know enough to know if that's correct or not, but it is somewhat at odds with your description.